tests/unittest_mlagent/mock_mlagent.h

```diff
@@ -62,7 +69,19 @@ MockModel *ml_agent_mock_get_model (const gchar *name, const guint version);
  * the following implementation is only for testing purposes.
  */
 gint ml_agent_model_get (
-    const gchar *name, const guint version, gchar **description, GError **err);
+    const gchar *name, const guint version, gchar **description);
+
+/**
+ * @brief Pass the JSON c-string generated by the ML Agent mock class to the caller.
+ * @param[in] name The model's name to retrieve
+ * @param[out] description The JSON c-string, containing the information of the model matching the given name and version
+ * @param[out] err The return location for a recoverable error. This can be NULL.
```

Comment:
Need to update doxygen tag
```suggestion
 * @param[out] description The JSON c-string, containing the information of the activated model matching the given name.
```

Reply:
Thanks! I updated those tags. PTAL